modules/ROOT/partials/misc/inline-formatting-of-list-bullets.adoc

```diff
@@ -0,0 +1,15 @@
+== Inline formatting.
+
+List bullets and list numbers can be spot- or inline-formatted.
+
+If an end-user selects the contents of a list item and applies inline formatting, such as a change of color or change of font-size, this spot formatting is also applied to the list item’s associated bullet or number.
+
+Only the bullets or list numbers associated with the selection are inline-formatted.
+
+NOTE: PowerPaste does not, currently, support inline-formatted bullets or list numbers. Inline-formatting, as applied to the bullets or list numbers, is lost when such material is copied and pasted.
+
+If the selection is one list item (ie, if the selection runs from an particular `<li>` tag to its associated `</li>` tag), only the bullet or number associated with the single selected list item takes on the inline-formatting.
```

Comment:
I'm legitimately surprised that you have `ie` here instead of `i.e.`, but you're far more likely to know :)

Reply:
Ah, the complicated relationship between the linguist’s knowledge of language change, the editor’s sense of what is and is not English formal tone, and the editor’s æsthetic.

_ie_ is now close to normative spelling in the various Commonwealth formal Englishes, including en-UK, en-AU, en-NZ, en-ZA, and en-CA. (A curious feature of this emergent normative spelling: it’s a rare example of a word that people resist capitalising the first letter of when the word is the first word in a sentence.)

When defences of this spelling are made, the common argument is that _ie_ is now a two-syllable English word that means _that is_.

Yes, it is a straight in-borrowing of the Latin abbreviation for _id est_, which happens to translate into English as _that is_.

But the large majority of extant users of this word have no idea of its Latin roots and just treat it as an English word that happens to be a perfect synonym for _that is_.

Curiously, en-US is clinging to the older spelling, _i.e_. This is curious because en-US, in the main, gets rid of non-character glyphs with what all other Englishes think of as unseemly haste (especially hyphens).

That said, the Commonwealth spelling is common in scientific en-US. And, by leakage and a shared ‘minimalist‘ æsthetic, it is also increasingly common in technical publishing.

My personal editorial æsthetic happens to align with the Commonwealth spelling of this (and its related Latin abbreviation, _eg_ for _exemplī grātiā_) so I’m happy to argue for technically-oriented material, such as our docs, to be a bit ahead of the general en-US editorial curve on this front because it puts us in line with the emerging norm of scientific and technical en-US.

All this said, your response is a reminder that there is a further argument that the plain English equivalents are even better. The original Latin abbreviations were affectations designed to mark class, and continue to be a source of confusion for most English speakers, especially ESOL folk.

I’ve edited accordingly.